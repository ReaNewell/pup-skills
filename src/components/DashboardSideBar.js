import React from 'react';
import { Link } from 'react-router-dom';

import DogList from './DogList';
import NameCard from './NameCard';

class DashboardSideBar extends React.Component {
    render() {
        return (
            <div className="dashboard-sidebar__container">
                <div className="dashboard-sidebar">
                    <NameCard />
                    <DogList />
                </div>
                <div>
                    <Link to='/settings'>Edit Profile</Link>
                </div>
            </div>
        )
    };
};

export default DashboardSideBar;