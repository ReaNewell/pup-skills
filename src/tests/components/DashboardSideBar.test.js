import React from 'react';
import { shallow } from 'enzyme';
import DashboardSideBar from '../../components/DashboardSideBar';

test('should render DashboardSideBar component correctly', () => {
    const wrapper = shallow(<DashboardSideBar />);
    expect(wrapper).toMatchSnapshot();
});