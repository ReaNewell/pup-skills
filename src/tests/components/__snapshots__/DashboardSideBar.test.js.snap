// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardSideBar component correctly 1`] = `
<div
  className="dashboard-sidebar__container"
>
  <div
    className="dashboard-sidebar"
  >
    <Connect(NameCard) />
    <Connect(DogList) />
  </div>
</div>
`;
